crates/protobuf_extractor/src/extract_protobuf_imports.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::Result;
+use tree_sitter::Parser;
+
+#[derive(Debug, PartialEq)]
+pub struct ProtobufSource {
+    pub imports: Vec<String>
+}
+
+pub fn parse(source: &str, _source_path: &str) -> Result<ProtobufSource> {
+    let mut buf = Vec::default();
+    let mut parser = Parser::new();
+    parser.set_language(tree_sitter_proto::language()).expect("tree-sitter failed to initialize");
```

Comment:
Should avoid the expect and can return the Err branch on the anyhow

Reply:
Yesterday I learned `?` - c48f242a73f03414855a3551ca71d70ebbd6e722